packages/travis-opam/travis-opam.1.0.2/opam

```diff
@@ -0,0 +1,7 @@
+opam-version: "1.2"
+maintainer:   "thomas@gazagnaire.org"
+homepage:     "https://github.com/ocaml/ocaml-travisci-skeleton"
```

Comment:
this has been renamed to `ocaml-ci-scripts` as it also contains some appveyor scripts to test with mingw